sdk/storage/test/test_base.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "gtest/gtest.h"
+
+namespace Azure
```

Comment:
namespaces should be in the same line

Reply:
I think format will be resolved with Clang formatter before release, no worries :)